requirements/requirements-dev.txt

```diff
@@ -3,7 +3,7 @@ click==8.0.4
 docker==5.0.3
 dredd-hooks==0.2.0
 flake8==3.8.4
-importlib-metadata<5  # need to pin <5 while flake8 is on 3.8.4
+importlib-metadata==8.5.0  # need to pin <5 while flake8 is on 3.8.4
```

Comment:
The latest version of otel package `opentelemetry-instrumentation-urllib`was conflicting with versions earlier than `importlib-metadata==8.5` so that had to be updated to 8.5. doesn't seem to affect checks using flake8. I will remove the comment.